package.json

```diff
@@ -15,22 +15,22 @@
         "eslint-import-resolver-node": "^0.3.6",
         "extend": "^3.0.2",
         "formik": "^2.2.9",
-        "gatsby": "^4.9.3",
+        "gatsby": "^4.14.1",
         "gatsby-background-image": "^1.6.0",
         "gatsby-plugin-anchor-links": "^1.2.1",
-        "gatsby-plugin-google-tagmanager": "^4.9.0",
-        "gatsby-plugin-image": "^2.9.1",
-        "gatsby-plugin-manifest": "^4.9.1",
+        "gatsby-plugin-google-tagmanager": "^4.14.0",
+        "gatsby-plugin-image": "^2.14.1",
```

Comment:
We will still get `.innerHTML` error with this version of `gatsby-plugin-image`.
For checking this you can go to /dmt5 page and see the console.

Reply:
This issue is resolved in this
"gatsby-plugin-image": "2.13.0",

Reply:
Thanks guys, downgraded  gatsby-plugin-image.